scripts/install.sh

```diff
@@ -239,11 +239,7 @@ remove_tmp_dirs() {
 create_sym_link() {
     log INFO "creating or overwriting sym link for :$1"
     # ln -sf source destination - creates symlink for dsc command to run from anywhere ex: any folder or location in the file system.
-    if [[ $OSTYPE == "msys"* ]]; then
```

Comment:
Sure. 

The main concern is security. If I want to install a script locally, I don't want to provide root permissions.

This change affect only this `if` clause - https://github.com/Azure/Bridge-To-Kubernetes/blob/731a8d9ecbdfdee3d7de2ca5a8abe4b067eb93cf/scripts/install.sh#L211-L219

So if we have `$HOME/.local/bin` in the `PATH` the script tries to install bridge in $HOME/.local/bin and create symlink there. In this case, you don't need root permissions to create symlink in $HOME/.local/bin (as you do for `cp` command a several line above).

I don't see how it would break `gitbash` because in case `"msys"` this command would be called without sudo

Reply:
@maksimr Thanks for the explanation. Yes, the above if condition takes care of ```msys```, I thought this script was using ```create_sym_link``` method in another place since there has been some changes recently. I will look further and confirm on this. 
next question - when you say root level permission for ```$HOME/.local/bin/``` is not needed are you talking about line ```chmod -R +x "$HOME/.local/bin/" ``` at line 214 ? because the if condition block doesn't contain any ```sudo```  statements , only the else condition has it. Please clarify on this part. 

![image](https://user-images.githubusercontent.com/105889062/227549848-3b811c2c-1bac-4ebf-b2c9-4c70503bf3e6.png)

Reply:
@maksimr I see your point, sometimes  ```chmod +x``` needs sudo in case of ```$HOME/.loca/bin``` if condition path correct ? If that's case we need to make sure it works without ```chmod +x```.

Reply:
> ... when you say root level permission for `$HOME/.local/bin/` is not needed are you talking about line `chmod -R +x "$HOME/.local/bin/" ` at line 214 ? 

Nope, only about using `sudo` command. `chmod -R +x` works fine in this case because current user usually owner of files (files in $HOME directory).

Reply:
Sorry I am still confused, with description of the PR and changes made. Can you please create an issue and link it to this PR, that would be of great help thanks.

Reply:
@maksimr any updates on this ??

Reply:
@maksimr since I didn't hear from you, I will go ahead and close this PR. Please feel free to open it again when you need them. Thank you.